easybuild/easyconfigs/g/gompi/gompi-2015b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2015b'
+
+homepage = '(none)'
```

Comment:
should we point to the easybuild readthedocs for toolchains here? None is also fine by me...